activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -777,7 +777,9 @@ def add_foreign_key(from_table, to_table, options = {})
         execute schema_creation.accept(at)
       end
 
-      # Removes the given foreign key from the table.
+      # Removes the given foreign key from the table. The options parameters will be
```

Comment:
I guess not really? My current implementation requires a `:from_table` and a `:to_table`, and throws `IrreversibleMigration` if both aren't provided. ([line in question](https://github.com/rails/rails/pull/20226/files#diff-117e4648370b1d1b9d5251b0e66c54aaR189)). But I suppose it would still work as long as the person is fine with having it be irreversible. I'll make that more clear.